conf/messages.en

```diff
@@ -336,9 +336,14 @@ legalChallenge.checkYourAnswersLabel = Do you know of any problems with the clas
 legalChallenge.error.required = Select yes if you know if there have been any problems classifying the item
 legalChallenge.paragraph = This includes court rulings and when a commodity code is being reviewed. This helps us to match the goods with other rulings.
 
-legalChallengeDetails.title = What are the problems with classifying your item? - Manage your Advance Tariff Rulings - GOV.UK
-legalChallengeDetails.heading = What are the problems with classifying your item?
+legalChallengeDetails.title = Provide details of the legal problems classifying the item - Manage your Advance Tariff Rulings - GOV.UK
```

Comment:
Should this be "problems classifying the goods" rather than "item"?

Reply:
Thanks for this, David. Checked with Eszter and have changed this and the other two "item" references in messages to "goods". Will merge when checks have passed.